optimum/exporters/openvino/model_configs.py

```diff
@@ -2773,3 +2805,274 @@ class MT5OpenVINOConfig(T5OpenVINOConfig):
 )
 class LongT5OpenVINOConfig(T5OpenVINOConfig):
     pass
+
+
+class JanusConfigBehavior(str, enum.Enum):
+    LANGUAGE = "language"
+    VISION_EMBEDDINGS = "vision_embeddings"
+    TEXT_EMBEDDINGS = "text_embeddings"
+    LM_HEAD = "lm_head"
+    VISION_GEN_EMBEDDINGS = "vision_gen_embeddings"
+    VISION_GEN_HEAD = "vision_gen_head"
+    VISION_GEN_DECODER = "vision_gen_decoder"
+
+
+class JanusDummyVisionGenInputGenerator(DummyInputGenerator):
+    SUPPORTED_INPUT_NAMES = ("pixel_values", "image_ids", "code_b", "image_shape", "lm_hidden_state", "hidden_state")
+
+    def __init__(
+        self,
+        task: str,
+        normalized_config: NormalizedConfig,
+        batch_size: int = DEFAULT_DUMMY_SHAPES["batch_size"],
+        sequence_length: int = DEFAULT_DUMMY_SHAPES["sequence_length"],
+        **kwargs,
+    ):
+        self.task = task
+        self.batch_size = batch_size
+        self.sequence_length = sequence_length
+        self.normalized_config = normalized_config
+
+    def generate(self, input_name: str, framework: str = "pt", int_dtype: str = "int64", float_dtype: str = "fp32"):
+        if input_name == "pixel_values":
+            return self.random_float_tensor(
+                [
+                    self.batch_size,
+                    1,
+                    3,
+                    self.normalized_config.config.params.image_size,
+                    self.normalized_config.config.params.image_size,
+                ]
+            )
+
+        if input_name == "image_ids":
+            return self.random_int_tensor(
+                [self.sequence_length],
+                max_value=self.normalized_config.config.params.image_token_size,
+                framework=framework,
+                dtype=int_dtype,
+            )
+        if input_name == "code_b":
+            return self.random_int_tensor(
+                [self.batch_size, 576],
+                max_value=self.normalized_config.config.params.image_token_size,
+                framework=framework,
+                dtype=int_dtype,
+            )
+        if input_name == "image_shape":
+            import torch
+
+            return torch.tensor(
+                [self.batch_size, self.normalized_config.config.params.n_embed, 24, 24], dtype=torch.int64
```

Comment:
could you add a comment to explain where 24 is coming from ?

Reply:
I used such values during debugging, replaced with calculation used config